doc/rules/function_notation/use_arrow_functions.rst

```diff
@@ -13,12 +13,26 @@ Using this rule is risky
 Risky when using ``isset()`` on outside variables that are not imported with
 ``use ()``.
 
+Configuration
+-------------
+
+``one_liners_only``
```

Comment:
what is the case we should not convert to arrow function if there is only one statement, the `return`, regardless how many lines it takes?

Reply:
BC promise? Current description mentions "one-liner", are we ok to drop that limit without config option?

Reply:
I would love to better understand if there is a need for anyone to handle single-statement but multi-line differently.

if not, no need for option and we simply extended the fixer to touch case it was not touching yet - it's same style, applied for more valid elements.

if yes, ok to keep option.

I'm also OK to have it behind toggle and put v4 TODO to remove it